test_requests/bicycle_routes.txt

```diff
@@ -1,6 +1,6 @@
 -j '{"locations":[{"lat":40.251919,"lon":-76.707581,"street":""},{"lat":40.262821,"lon":-76.685020,"street":"Life Lion Drive"}],"costing":"bicycle","directions_options":{"units":"miles"}}'
--j '{"locations":[{"lat":40.07334267155828,"lon":-75.31021356669953,"type":"break"},{"lat":40.11710395215845,"lon":-75.35510301677277,"type":"break"}],"costing":"bicycle","costing_options":{"bicycle":{"bicycle_type":"Road","use_roads":"0.5","cycling_speed":"25.0","use_hills":"0.0"}}}'
--j '{"locations":[{"lat":40.72280306615735,"lon":-74.04238701215945,"type":"break"},{"lat":40.7196806451724,"lon":-74.0063381229993,"type":"break"}],"costing":"bicycle","costing_options":{"bicycle":{"bicycle_type":"Road","use_roads":"0.5","cycling_speed":"25.0","use_hills":"0.0"}}}'
+-j '{"locations":[{"lat":40.07334267155828,"lon":-75.31021356669953,"type":"break"},{"lat":40.11710395215845,"lon":-75.35510301677277,"type":"break"}],"costing":"bicycle","costing_options":{"bicycle":{"bicycle_type":"road","use_roads":"0.5","cycling_speed":"25.0","use_hills":"0.0"}}}'
```

Comment:
Only some here were changed to be lowercased.
Seems amateur-ish.. What do you think? (Idk if this is the best approach or if I should do something different)

Reply:
these requests are for testing the output really, for the purpose of this change i would leave this file untouched. can you please revert it?